### PR TITLE
extend intro and add summary of version history

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

**Comment by ablwr** [comment]
👍

File: rfc_frontmatter.md [hunk 1]
```diff
@@ -156,4 +156,28 @@ This document defines FFV1, a lossless intra-frame video encoding format. FFV1 i
...
+    <author initials="M." surname="Niedermayer" fullname="Michael Niedermayer"/>
+    <date month="April" year="2009" />
+  </front>
+</reference>
+
+<reference anchor="FFV1_V3" target="https://git.libav.org/?p=libav.git;a=commit;h=fe6e5cbea7dbd5d2c67d79b5570e26debb70e95b">
+  <front>
+    <title>Commit to mark FFV1 version 3 as non-experimental</title>
+    <author initials="L." surname="Barbato" fullname="Luca Barbato"/>
+    <date month="June" year="2016" />
```
Comment (michaelni) [inline rfc_frontmatter.md]: This is incorrect, in fact all of the 1.3 entry
1.3 was made non experimental in 2013 by me
commit abe76b851c05eea8743f6c899cbe5f7409b0f301
Author: Michael Niedermayer <michaelni@gmx.at>
Date:   Sat Aug 17 04:28:53 2013 +0200

    ffv1enc: Make ffv1.3 non experimental

**Comment by dericed** [comment]
ooops. Updated to correct info via https://github.com/FFmpeg/FFV1/pull/72#discussion_r121244033. Please re-review.

**Comment by dericed** [comment]
ping. any other changes suggested?

**Comment by retokromer** [comment]
LGTM

**Comment by michaelni** [comment]
build seems to fail here: (seems a simple typo, ill fix and push)


Parsing file draft-ietf-cellar-ffv1-02.xml
WARNING: Parsing Error: error parsing attribute name, line 1533, column 1
ERROR: Unable to parse the XML document: draft-ietf-cellar-ffv1-02.xml
 draft-ietf-cellar-ffv1-02.xml: Line 1533: error parsing attribute name
 draft-ietf-cellar-ffv1-02.xml: Line 1533: attributes construct error
 draft-ietf-cellar-ffv1-02.xml: Line 1533: Couldn't find end of Start Tag author
 draft-ietf-cellar-ffv1-02.xml: Line 1533: error parsing attribute name
 draft-ietf-cellar-ffv1-02.xml: Line 1533: attributes construct error
 draft-ietf-cellar-ffv1-02.xml: Line 1533: Couldn't find end of Start Tag author line 1532
make: *** [draft-ietf-cellar-ffv1-02.html] Error 1

**Pull request closed without merging**